fitIn/popUpViewController.swift

```diff
@@ -10,19 +10,21 @@ import UIKit
 
 class popUpViewController: UIViewController {
 
-   private var scenarioController = ScenarioHandler()
```

Comment:
where did this go? is it not used?

Reply:
I dont neeed it in the pop up view, just the original scenario handler view. The way I have it set up is the scenario handler passes the answer reason to the pop up view